src/Artifactory/ArtifactoryClient.ts

```diff
@@ -8,7 +7,7 @@ import { IClientSpecificConfig } from '../../model/ClientSpecificConfig';
 
 export class ArtifactoryClient {
     private readonly httpClient: HttpClient;
-    private logger: ILogger;
+    private logger: ArtifactoryLogger;
```

Comment:
I suggest using the ILogger interface and hiding the implementation details:
```suggestion
    private logger: ILogger;
```
For more information see: https://stackoverflow.com/questions/66418160/understanding-coding-to-an-interface-to-hide-implementation-details
Same in XrayLogger.